scan/__tests__/Route_test.re

```diff
@@ -0,0 +1,50 @@
+open Jest;
+open Route;
+open Expect;
+
+describe("Expect Search Functionality to work correctly", () => {
+  test("test block route", () =>
+    expect("123" |> search |> toString) |> toEqual("/block/123")
+  );
+  test("test block route", () =>
+    expect(
+      "22638794cb5f306ef929b90c58b27d26cb35a77ca5c5c624cf2025a98528c323" |> search |> toString,
+    )
+    |> toEqual("/tx/22638794cb5f306ef929b90c58b27d26cb35a77ca5c5c624cf2025a98528c323")
+  );
+  test("test block prefix is B", () =>
```

Comment:
Let's have this compared route directly if possible? We should not have tests that rely on `toString` unless necesssary.